src/containers/BooksForm.js

```diff
@@ -1,31 +1,45 @@
 import PropTypes from 'prop-types';
 import { useState } from 'react';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
 
-const BooksForm = props => {
-  const { categories } = props;
-  const [book, setBook] = useState({});
+const BooksForm = ({ createBook }) => {
+  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  const bookReset = { title: '', category: 'Category' };
+  const [book, setBook] = useState(bookReset);
   const handleInputChange = event => {
     setBook({
+      ...book,
       title: event.target.value,
     });
   };
 
   const handleSelectChange = event => {
     setBook({
+      ...book,
       category: event.target.value,
     });
   };
 
```

Comment:
You can refactor your codebase to use a single function to handlechange for both input and select element.


``` 
handleChange({ target: { name, value } }) {
  this.setState({
    [name]: value,
  })
}
 ```

... then pass it as a prop for **onChange event** to input and select elements